app/Controllers/configureController.php

```diff
@@ -358,9 +358,6 @@ public function systemAction() {
 			Minz_Error::error(403);
 		}
 
-		$can_enable_email_validation = version_compare(PHP_VERSION, '5.5') >= 0;
-		$this->view->can_enable_email_validation = $can_enable_email_validation;
```

Comment:
Not currently relevant of course, but I think we were still going for 5.3+ when this one was added (and as such it's not equivalent with min version since it's just the one feature).

Reply:
I know but now, the minimum requirement is 5.6. So there is no need to keep a test that will be always true.